.github/workflows/publish-images.yaml

```diff
@@ -0,0 +1,48 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    tags:
+      - "*"
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [dask-gateway, dask-gateway-server]
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
```

Comment:
I understand this step to extract details about the version based on the pushed git tag that triggered this workflow. But, it is never used. If it was, we would see `steps.get_version.outputs.VERSION` be referenced somewhere.